src/components/sdk/ContentShare/index.tsx

```diff
@@ -10,37 +10,49 @@ import { BaseSdkProps } from '../Base';
 
 interface Props extends BaseSdkProps {
   nameplate?: string;
+  tileId?: number;
 }
 
 export const ContentShare: React.FC<React.PropsWithChildren<Props>> = ({
   className,
+  tileId,
   ...rest
 }) => {
   const audioVideo = useAudioVideo();
-  const { tileId } = useContentShareState();
+  const contentShareState = useContentShareState();
```

Comment:
Thinking from keeping the props limited with no new optional props idea. How about we maintain `tiles` as an array in the content share state. Based on the number of tiles, we should show those many `ContentTile`s?

Reply:
But yes, we would have to maintain support of `contantShareState.tileId`.

Reply:
In short, I decide to let builder to decide how and where to render the multiple content share tiles via the new `tileId` of `ContentShare` component. Check code example in its documentation for example.